artemis-distribution/src/main/resources/licenses/bin/LICENSE

```diff
@@ -432,4 +432,46 @@ For Underscore:
 ==============================================================================
 This product bundles Underscore, which is available under a
 "MIT" license.  For details, see
-https://github.com/jashkenas/underscore/blob/master/LICENSE
\ No newline at end of file
+https://github.com/jashkenas/underscore/blob/master/LICENSE
+
+==============================================================================
+For jakarta.activation-api:
+==============================================================================
+This product bundles Underscore, which is available under the
```

Comment:
The entry is for jakarta.activation-api but it mentions including "Underscore", which was the entry above.

Same issue for all the entries that follow.

Reply:
fixed